web/views/tracker_view.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Healthlocker.TrackerView do
   use Healthlocker.Web, :view
   use Timex
+  import Healthlocker.ComponentView, only: [markdown: 1]
```

Comment:
We may need to require in another markdown function -> this one in particular does some extra work for adding links to the hashtags in tips